x-pack/qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/RollupIDUpgradeIT.java

```diff
@@ -89,7 +89,7 @@ public void testIDsUpgradeCorrectly() throws Exception {
             client().performRequest(indexRequest);
 
             // create the rollup job
-            final Request createRollupJobRequest = new Request("PUT", "/_xpack/rollup/job/rollup-id-test");
+            final Request createRollupJobRequest = new Request("PUT", "/_rollup/job/rollup-id-test");
```

Comment:
I'm not entirely sure how rolling upgrade tests pick their versions, but couldn't we go from `< 6.6` to `>= 6.6, <7.0`?  In which case we'll need an `assumeTrue()` on this test, or some kind of version check on these URIs?

Reply:
The rolling upgrade tests only go through the wire compatible versions so this should be safe.